assets/src/scripts/components/graph/state/options.js

```diff
@@ -0,0 +1,50 @@
+import { createSelector } from 'reselect';
+
+import { getWaterLevelID } from 'ngwmn/services/state/index';
+
+const MOUNT_POINT = 'components/graph/options';
+const GRAPH_OPTIONS_SET = `${MOUNT_POINT}/GRAPH_OPTIONS_SET`;
+
+
+/**
+ * Action creator to set current options for the graph.
+ * @param {Object} options Graph options object
+ */
+export const setOptions = function (options) {
+    return {
+        type: GRAPH_OPTIONS_SET,
+        payload: {
+            options
```

Comment:
Yes it should be explicit rather than store whatever is passed to it. 👍 

I disagree on the ownership.. With the state normalized by ID (cache store state is like `{"USGS:01000000": {.....}` the graph needs to know what data to request in the selector, and since it's specified as an argument to the graph constructor, that seems like the right place to me. In theory, this would allow us to display multiple site graphs on the same page.

Reply:
How about `setGraphOptions(agencyCode, siteID)`?

Reply:
I like the name. I can see your point and once I had gone through the whole pull request, I realized that's why you did this and that we should be able to handle multiple sites so I'm fine leaving it